aten/src/ATen/native/Pool.h

```diff
@@ -191,6 +191,12 @@ max_pool2d_backward_shape_check(
   check_dim_size(indices, ndim, ndim-3, nOutputPlane);
   check_dim_size(indices, ndim, ndim-2, outputHeight);
   check_dim_size(indices, ndim, ndim-1, outputWidth);
+
+  if (ndim == 4) {
+    const int64_t BatchSize = input.size(0);
```

Comment:
Please use `cameCase` (or actually `snake_case`)
```suggestion
    const int64_t batchSize = input.size(0);
```